internal/cli/login/aap.go

```diff
@@ -58,16 +62,19 @@ func (o AAPOAuth) getOAuth2Client(callback string) (*osincli.Client, error) {
 		SendClientSecretInParams: true,
 		RedirectUrl:              callback,
 		Scope:                    "read",
```

Comment:
we should take the scopes as a space delimited string from the output of /auth/config ( see providers[x].spec.scopes for oidc/oauth2/openshift